resotocore/resotocore/config/__init__.py

```diff
@@ -17,6 +19,66 @@ class ConfigEntity:
     config: Json
     revision: Optional[str] = None
 
+    def analytics(self) -> Dict[str, Any]:
+        analytics: Dict[str, Any] = {"config_id": self.id}
+        if "resotoworker" not in self.config:
+            return analytics
+
+        # provider information
+        collectors: List[str] = []
+        collectors.extend(cast(List[str], value_in_path(self.config, ["resotoworker", "collector"])))
+        if "example" in collectors:
+            collectors.remove("example")
+        analytics.update({"how_many_providers": len(collectors)})
+
+        # authentication information
+        if "aws" in collectors:
+            analytics.update({"aws": True})
+
+            aws_use_access_secret_key = (
+                value_in_path(self.config, ["resotoworker", "aws", "access_key_id"]) is not None
+                and value_in_path(self.config, ["resotoworker", "aws", "secret_access_key"]) is not None
+            )
+            aws_use_role = value_in_path(self.config, ["resotoworker", "aws", "role"]) is not None
+            aws_use_profiles = value_in_path(self.config, ["resotoworker", "aws", "profiles"]) is not None
+            aws_use_accounts = value_in_path(self.config, ["resotoworker", "aws", "account"]) is not None
+            aws_use_scrape_org = value_in_path(self.config, ["resotoworker", "aws", "scrape_org"])
+
+            analytics.update({"aws_use_access_secret_key": aws_use_access_secret_key})
```

Comment:
```suggestion
            analytics["aws_use_access_secret_key"] = aws_use_access_secret_key
```
This way no intermediate objects are created and is simpler to read?
True for all other `update` calls.